### PR TITLE
Finalizing JSON monitoring for 2016

### PR DESCRIPTION
**Pull request by avetisya**
81X version of #14030 . It's completely identical to the 80X version.

### CODE REVIEW

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
+1

File: HLTrigger/JSONMonitoring/plugins/L1TriggerJSONMonitoring.cc [hunk 1]
```diff
@@ -0,0 +1,452 @@
...
+{
+  std::shared_ptr<l1Json::lumiVars> iSummary(new l1Json::lumiVars);
+
+  unsigned int MAXPATHS = 512;
+
+  iSummary->processed = new HistoJ<unsigned int>(1, 1);
+
+  iSummary->prescaleIndex = 100;
+
+  iSummary->L1AlgoAccept            = new HistoJ<unsigned int>(1, MAXPATHS);
```
Comment (davidlange6) [inline HLTrigger/JSONMonitoring/plugins/L1TriggerJSONMonitoring.cc]: how do these pointers get deleted?

Comment (avetisya) [inline HLTrigger/JSONMonitoring/plugins/L1TriggerJSONMonitoring.cc]: The pointers are deleted in the L1TriggerJSONMonitoring::globalEndLuminosityBlockSummary function starting on line 393.

Comment (davidlange6) [inline HLTrigger/JSONMonitoring/plugins/L1TriggerJSONMonitoring.cc]: thanks - they should certainly also be reinitialized to nullptr. Please add a new PR for that. Thanks.

**Comment by davidlange6** [comment]
+1

**Pull request merged**